pyanaconda/modules/payloads/source/harddrive/initialization.py

```diff
@@ -51,8 +57,8 @@ def run(self):
         order again.
 
         :raise: SourceSetupError
-        :return: path to the install tree
-        :rtype: str
+        :return: named tuple with path to the install tree and name of ISO if set or empty string
+        :rtype: namedtuple("SetupHardDriveResult", ["install_tree_path", "iso_name"])
```

Comment:
I think it should be enough to specify the type as an instance of `SetupHardDriveResult`.

Reply:
Fixed.